db/migrate/20221104112658_add_more_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddMoreToUsers < ActiveRecord::Migration[7.0]
```

Comment:
There is a migration error so I could not run the app. Kindly make sure the database is created and migrated successfully.
![mig](https://user-images.githubusercontent.com/35504149/199996487-9ea93bc0-3c24-455a-99fb-b4a5af72418a.png)

Reply:
okay thank you so much 😄